tests/test_qubit_device.py

```diff
@@ -123,90 +147,96 @@ def test_op_queue_accessed_outside_execution_context(self, mock_qubit_device):
         """Tests that a call to op_queue outside the execution context raises the correct error"""
 
         with pytest.raises(
-            ValueError, match="Cannot access the operation queue outside of the execution context!"
+                ValueError, match="Cannot access the operation queue outside of the execution context!"
```

Comment:
Was this a result of running `black`?

Reply:
This is so strange! Must be...let me change it back!